automatic/jenkins/jenkins.nuspec

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+    <metadata>
+        <id>jenkins</id>
+        <version>2.222.4</version>
+        <packageSourceUrl>https://github.com/git@github.com:chocolatey-community/chocolatey-packages/tree/master/automatic/jenkins</packageSourceUrl>
+        <owners>pauby,johnypony3</owners>
```

Comment:
The `chocolatey-community` user should be listed as the first entry in the owners element.

https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md#124-add-chocolatey-community-among-owners